tests/integration_tests.rs

```diff
@@ -0,0 +1,141 @@
+use std::{fs::File, path::Path, process::Command};
+use tempfile::tempdir;
+
+use flate2::write::GzEncoder;
+use flate2::Compression;
+
+#[test]
+fn integration_tests() {
+    let package_name = "pulldown-cmark";
+    let wapm_out = Command::new("cargo")
+        .args(["wapm", "--dry-run"])
+        .output()
+        .expect("Failed to execute command");
+
+    assert!(wapm_out.status.success(), "{wapm_out:?}");
+
+    let temp_dir = tempdir().unwrap();
+
+    let project_dir_path = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let wapm_dir = project_dir_path.join(format!("target/wapm/{}", package_name));
+
+    assert!(
+        wapm_dir.exists(),
+        "{}",
+        format!("wapm-dir for {} not found", package_name)
+    );
+
+    // Create tar.gz
+    let tar_gz = File::create(temp_dir.path().join(format!("{}.tar.gz", package_name))).unwrap();
+    let enc = GzEncoder::new(tar_gz, Compression::default());
+    let mut tar = tar::Builder::new(enc);
+    tar.append_dir_all(".", &wapm_dir).unwrap();
+    tar.into_inner()
+        .expect("Unable to finalise the tar archive")
+        .finish()
+        .expect("Unable to finalize the gzip encoder");
```

Comment:
The `wasmer-pack` command can work with directories, so we don't need to generate this tarball or run `wapm2pirita`.

Just run `wasmer run wasmer-pack --mapdir .:$WAPM_DIR -- python . --out-dir ./python`, where `$WAPM_DIR` is the path to your `target/wapm/` directory.